src/client/stylesheets/style.less

```diff
@@ -339,6 +339,11 @@ a.contact-text:visited {
 
 /* react-datepicker styling */
 .form-group {
+	.res-dob{
```

Comment:
A more descriptive name for this class would be better. I have no idea what this means 😅 
res(ponsive)-somethingsomething?

Reply:
responsive date of birth ...

Reply:
I'd rather see the long form .responsive-date-field or something rather than the cryptic .res-dob